pkg/config/config.go

```diff
@@ -444,3 +447,20 @@ func SetCustomRuleConfigMap(hasConfigMap bool) {
 func HasCustomRuleConfigMap() bool {
 	return hasCustomRuleConfigMap
 }
+
+func GetCertDuration() time.Duration {
+	return certDuration
+}
+
+func SetCertDuration(annotations map[string]string) {
+	if annotations != nil && annotations[AnnotationCertDuration] != "" {
+		d, err := time.ParseDuration(annotations[AnnotationCertDuration])
+		if err != nil {
+			log.Error(err, "Failed to parse cert duration, use default one", "annotation", annotations[AnnotationCertDuration])
+		} else {
+			certDuration = d
+			return
+		}
+	}
+	certDuration = time.Hour * 24 * 365
```

Comment:
I think we can remove this line. because `certDuration ` default is `time.Hour * 24 * 365`.

Reply:
It's my fault. it should override the default. Got it.